src/utils/subsonic_json_parser.cpp

```
@@ -0,0 +1,81 @@
+
+#include <string>
+
+namespace subsonic {
+namespace json_parser {
+
+pfc::string8 to_string(const nlohmann::json &value) {
+	if (value.is_string()) {
+		return value.get_ref<const std::string &>().c_str();
+	}
+	if (value.is_number_integer()) {
+		return std::to_string(value.get<long long>()).c_str();
```

Cppcheck (2.20.0)

[error] Returning object that will be invalid when returning.

(returnDanglingLifetime)